west.yml

```diff
@@ -174,7 +174,8 @@ manifest:
       groups:
         - crypto
     - name: mcuboot
-      revision: e86f575f68fdac2cab1898e0a893c8c6d8fd0fa1
+      url: https://github.com/mcu-tools/mcuboot
```

Comment:
such reference might be used only for testing purpose. We are using zephyr-rtos/mcuboot fork here.
I will make its update right after the 3.1.0 release.

Reply:
@nvlsianpu 

Ok, thanks for info. Update to west.yml has been removed from PR.